lib/job-simulation.rb

```diff
@@ -2,19 +2,47 @@
 require './Queue.rb'
 
 class JobSimulation
-  attr_reader :workers, :waiting, :roll
+  #attr_reader :workers, :waiting, :roll
 
-  def initialize (jobs_available, job_seekers)
+  def initialize (job_seekers)
```

Comment:
You should not change the definition of creating a JobSimulation (changing the number of parameters). When you are given a definition to complete, method names and parameter types/numbers should not be altered unless specifically told to do so.